pymatgen/entries/compatibility.py

```diff
@@ -463,12 +463,13 @@ def get_correction(self, entry) -> ufloat:
         """:param entry: A ComputedEntry/ComputedStructureEntry
 
         Returns:
-            Correction, Uncertainty.
+            ufloat: Energy correction in eV and uncertainty of the correction in eV.
         """
-        if entry.parameters.get("run_type") not in ("GGA", "GGA+U"):
-            raise CompatibilityError(
-                f"Entry {entry.entry_id} has invalid run type {entry.parameters.get('run_type')}. Discarding."
-            )
+        # Only correct GGA or GGA+U entries
+        run_type = entry.parameters.get("run_type")
+        if run_type not in ("GGA", "GGA+U"):
+            warnings.warn(f"Entry {entry.entry_id} has invalid {run_type=}. Applying no corrections.")
+            return ufloat(0.0, 0.0)
```

Comment:
This will need testing, but I'm fairly certain this entire block should be deleted. The `get_corrections` methods should not raise `CompatibilityError` anymore; rather they should just return `ufloat(0,0)` for any entries to which the correction does not apply.

`CompatibilityError` is raised by `get_adjustments`